make.go

```diff
@@ -331,9 +335,11 @@ func getDebianEmail() string {
 	if email := strings.TrimSpace(os.Getenv("DEBEMAIL")); email != "" {
 		return email
 	}
-	if mailname, err := ioutil.ReadFile("/etc/mailname"); err == nil {
-		if u, err := user.Current(); err == nil {
-			return u.Name + "@" + strings.TrimSpace(string(mailname))
+	mailname, err := ioutil.ReadFile("/etc/mailname")
+	if err == nil && strings.Contains(string(mailname), ".") {
+		// By default, /etc/mailname contains "debian" which is not useful; check for ".".
```

Comment:
nit: the comment belongs one line higher up in the file, before the check that checks for "."